src/components/ColorList/ColorList.module.css

```diff
@@ -0,0 +1,12 @@
+.item{
+    padding: 3px;
+    border: solid 1px #517B97;
+    outline: 0;
+    background: -webkit-gradient(linear, left top, left 25, from(#FFFFFF), color-stop(4%, #CAD9E3), to(#FFFFFF));
```

Comment:
There are 2 background properties, it isn't needed. And also there is one extra thing. I think you should delete prefixes -webkit-gradient, -moz-linear-gradient, things like that. These prefixes will be added by Webpack.